package.json

```diff
@@ -32,6 +32,8 @@
     "@nestjs/serve-static": "^4.0.0",
     "@nestjs/swagger": "^7.1.7",
     "@node-redis/json": "^1.0.2",
+    "@openzeppelin/contracts": "^4.9.3",
```

Comment:
I'm still considering using the `transfer` ABI item for ERC20 contracts. Including a whole collection of contracts in the project does not seem necessary. 2Mb are added to the docker image with this dependency.

Reply:
As the contracts are immutable, I see no issue in not installing this package but we should analyse the bundle to see if it's "tree shakeable" first.

Reply:
I removed the dependency since we don't strip the unused assets of this package on the final image. Since the scope is limited to the `transfer` call I decided to limit its scope (i.e.: the ABI only has `transfer`). [532f5ad](https://github.com/safe-global/safe-client-gateway/pull/680/commits/532f5ad1ede95d7d1e9ce70f2334db02ec293e1e)